src/cuga/backend/server/manage_routes/knowledge_reindex.py

```
@@ -0,0 +1,180 @@
+
+    # Spawn the deferred pointer-flip. The HTTP response returns NOW with
+    # task_ids so the UI shows progress immediately; the pointer flips
+    # behind the scenes once workers terminate AND the engine config
+    # still matches target_hash. See ``deferred_reindex_complete_and_flip``.
+    task_ids = (r or {}).get("task_ids") or []
+    if target_hash and live_state is not None and task_ids:
+        _asyncio.create_task(
+            deferred_reindex_complete_and_flip(
+                agent_id, live_engine, live_state, target, target_hash, task_ids
+            )
+        )
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)